lib/src/components/NxTransferListHalf/__tests__/NxTransferListHalf.test.tsx

```diff
@@ -4,495 +4,537 @@
-      const items = [{
-            id: 1,
-            displayName: 'top'
-          }, {
-            id: 3,
-            displayName: 'middle'
-          }, {
-            id: 4,
-            displayName: 'bottom'
-          }],
-          component = getMounted({ allowReordering: true, items }),
-          list = component.find('.nx-transfer-list__item-list');
-
-      const firstItem = list.find('.nx-transfer-list__item').at(0),
-          secondItem = list.find('.nx-transfer-list__item').at(1),
-          thirdItem = list.find('.nx-transfer-list__item').at(2),
-          firstItemMoveUpButton = firstItem.find('.nx-btn').at(0),
-          firstItemMoveDownButton = firstItem.find('.nx-btn').at(1),
-          secondItemMoveUpButton = secondItem.find('.nx-btn').at(0),
-          secondItemMoveDownButton = secondItem.find('.nx-btn').at(1),
-          thirdItemMoveUpButton = thirdItem.find('.nx-btn').at(0),
-          thirdItemMoveDownButton = thirdItem.find('.nx-btn').at(1);
-
-      expect(firstItem).toExist();
-      expect(firstItemMoveUpButton).toExist();
-      expect(firstItemMoveDownButton).toExist();
-      expect(firstItemMoveUpButton).toHaveClassName('disabled');
-      expect(firstItemMoveDownButton).not.toHaveClassName('disabled');
-
-      expect(secondItem).toExist();
-      expect(secondItemMoveUpButton).toExist();
-      expect(secondItemMoveDownButton).toExist();
-      expect(secondItemMoveUpButton).not.toHaveClassName('disabled');
-      expect(secondItemMoveDownButton).not.toHaveClassName('disabled');
-
-      expect(thirdItem).toExist();
-      expect(thirdItemMoveUpButton).toExist();
-      expect(thirdItemMoveDownButton).toExist();
-      expect(thirdItemMoveUpButton).not.toHaveClassName('disabled');
-      expect(thirdItemMoveDownButton).toHaveClassName('disabled');
+    beforeEach(function() {
+      // fixes "An update to ForwardRef(Tooltip) inside a test was not wrapped in act" errors
+      jest.useFakeTimers();
     });
-  });
-
-  it('passes onReorderItem with id and direction when move up or down button is clicked', function() {
-    const items = [{
-      id: 1,
-      displayName: 'a'
-    }, {
-      id: 2,
-      displayName: 'b'
-    }, {
-      id: 3,
-      displayName: 'c'
-    }];
-
-    const onReorderItem = jest.fn(),
-        component = getMounted({ allowReordering: true, items, onReorderItem }),
-        list = component.find('.nx-transfer-list__item-list');
-
-    expect(onReorderItem).not.toHaveBeenCalled();
 
-    const firstItem = list.find('.nx-transfer-list__item').at(0),
-        secondItem = list.find('.nx-transfer-list__item').at(1);
-    const moveDownButton = firstItem.find('.nx-btn').at(1);
-    const moveUpButton = secondItem.find('.nx-btn').at(0);
+    afterEach(runTimers);
 
-    expect(moveDownButton).toExist();
-    expect(moveUpButton).toExist();
+    async function runTimers() {
```

Comment:
This is going to come up on every component that has tooltips.  Let's move these to a file of utility functions

Reply:
> This is going to come up on every component that has tooltips. Let's move these to a file of utility functions

Done in RSC-1336

Reply:
> > This is going to come up on every component that has tooltips. Let's move these to a file of utility functions
> 
> Done in RSC-1336

Merged with main here: [5eed414](https://github.com/sonatype/sonatype-react-shared-components/pull/880/commits/5eed41445b5e905b6c731b3ef177428f780c46e1) and removed redundant code here: [61e13d4](https://github.com/sonatype/sonatype-react-shared-components/pull/880/commits/61e13d48b2a6541ff75e5394972af6c54badf465). Thanks for working on this in RSC-1336